TACC/tup-ui
# Task/tup 443 empty placeholder component for tickets projects

## Overview:
Created new empty table component for projects and tickets to use when no projects or tickets exist for a user

## Related:

- [TUP-443](https://jira.tacc.utexas.edu/browse/TUP-443)

## Changes:

- Refactored project empty table piece and created new component in utils for empty tables

- Created style sheet in Utils dir


## Testing:

1. Go to projectlisting.tsx and remove data from line 14
2. Create a new line under line 14 and on line 15 put
`const { data } = {}`
3. Go to project listing page and make sure the error box shows up with link to create a new project
4. Do the same steps for ProjectsTable.tsx this time removing data from line 20 and on a new line 21 put
`const { data } = {}`
5. Go to dashboard and make sure the projects shows the right alert for no projects
6. Go to TicketsTable.tsx and remove data from line 44
7.  On a new line 45, put
'const { data } = {}`
8. Make sure the ticket panel on the dashboard shows the right message (should not have a link)
9. Go to the ticket listing page and make sure the table has the right message 

## UI:
TIcket dashboard
<img width="1415" alt="Screen Shot 2023-03-15 at 5 17 22 PM" src="https://user-images.githubusercontent.com/96220951/225455984-8f65fc8a-0727-4f11-9f71-cc8a2fd28aa7.png">

Ticket listing
<img width="1770" alt="Screen Shot 2023-03-15 at 5 17 44 PM" src="https://user-images.githubusercontent.com/96220951/225456031-f0168dd8-5b38-4eba-a3a7-56008a2e80d9.png">

Project Dashboard
<img width="1553" alt="Screen Shot 2023-03-15 at 5 18 33 PM" src="https://user-images.githubusercontent.com/96220951/225456147-c2132216-0e1e-4a87-9409-c7d1dc4e61c8.png">

Project Listing
<img width="1577" alt="Screen Shot 2023-03-15 at 5 19 32 PM" src="https://user-images.githubusercontent.com/96220951/225456301-d77a7ffb-fdf2-4323-be14-39dc283b905f.png">


## Notes:
